scripts/qa_crowbarsetup.sh

```diff
@@ -3143,6 +3143,12 @@ function deploy_single_proposal
         ceph)
             [[ $deployceph ]] || return
             ;;
+        ceilometer)
+            if [[ $want_monasca_proposal != 1 ]]; then
```

Comment:
This will dosable ceilometer also for older cloud versions (where we want to still test ceilometer)

Reply:
done, thank you for spotting.